content/en/blog/_posts/2022-10-18-kubernetes-1.26-deprecations-and-removals.md

```diff
@@ -117,9 +117,9 @@ command line arguments relating to logging. These command line arguments were
 already deprecated.
 For more information, see [Deprecate klog specific flags in Kubernetes Components](https://github.com/kubernetes/enhancements/tree/3cb66bd0a1ef973ebcc974f935f0ac5cba9db4b2/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components).
 
-## Looking ahead {#looking-ahead}
+## Looking ahead
```

Comment:
We should retain the looking-ahead bit to retain consistency with other headers of the same type.